sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * Fixed possible `NullPointerException` issue if health-check flow kicks in before RNTBD context negotiation for a given channel - See [PR 36397](https://github.com/Azure/azure-sdk-for-java/pull/36397). 
 
 #### Other Changes
+* Handling negative end-to-end timeouts provided more gracefully by throwing a `CosmsoException` (`OperationCancelledException`) instead of `IllegalArgumentException`. - See [PR 36535](https://github.com/Azure/azure-sdk-for-java/pull/36535)
```

Comment:
nit: typo `CosmsoException` -> `CosmosException`

Reply:
Fixed it in this PR : https://github.com/Azure/azure-sdk-for-java/pull/36542